Feature/form bsc

ponche opened the pull request:
mise en place de la logique de modal Retard train. 
et du formulaire BSC avec la sauvegarde. 

récupération des valeur déjà sauvegardé au moment de l’ouverture page ou du modal et injectée directement dans les champ des formulaire. 

reste à mettre en place les notifications avec un  SnackBar pour avertir l'utilisateur de la bonne sauvegarde ou de saisi incorrect.

ponche committed:
✏️ Correction fautes d'orthographes dans README

ponche committed:
🚸 (bsc): Ajout la valeur par défaut de modal RetardTrain

ajout la valeur par défaut dans le formulaire directement à l'ouverture de la modal RetardTrain

ponche committed:
👔 (bsc): Mise en place de la sauvegarde des questionnaire

mise en place de la sauvegarde des questionnaires sur la page SaisiBSC et une petit refactor sur l'affichage du retard train

ponche committed:
🚸 (bsc): Mise en place des valeur par défaut saisiBSC

mise en place des valeur par défaut du formulaire BSC lors de l'ouverture de la page SaisiBSC si des données sont déja enregistré

ponche committed:
✏️ Corrections des fautes d'orthographes divers

ajout des S oublié pour le hook useDispatchCourses et correction d'une nom de composant et fichier  SaisieBscSreen

Sergio-bzh reviewed (commented):
Bonjour Rémi,
Vraiment désolé mais j'ai l'impression de lire une une langue étrangère (probablement du elfique).

La seule chose que j'ai trouvé à redire est une erreur d'ortographe ; mais vu que ça concerne Bsc c'est à dire "SaisiBsc".

Cela implique beaucoup de changements dans le projet et du coup je ne suis pas sûr que pour une "e" manquante on se donne tant de boulot :-(

SaisiBsc =  SaisieBsc (manque un e)
Saisi E BscScren = SaisieBscScreen (manque deux e)

A toi de voir.

Sergio

ponche committed:
🚸 (bsc): Change le clavier de saisi de SaisieBscSreen

change le clavier de saisi de type number-pad afin que l'utilisateur ne puisse rentrer que des  nombre

ponche commented:
Bon, j'ai regardé et mon commit [df8693d](https://github.com/gestionEnqueteur/app-gestionEnqueteur/pull/44/commits/df8693d122c2ad6f191d7361f90a8188f93ded94) a tous cassé mon application. 

j'avais pas rajouté les S à tous les fichiers. 

autre point que j'avais oublié, c'est que j'avais oublié de changé le type de clavier pour la saisi du nombre de questionnaire. 
c'est plus facile d'écrire un nombre avec un clavier numérique.

@Sergio-bzh , concernant le langage elfique, tu finira par l'apprendre 😄

jeanCodeUnMax commented on pages/bsc/SaisieBscScreen.tsx:
vide me donne de la confusion  !

jeanCodeUnMax commented on pages/bsc/SaisieBscScreen.tsx:
th arrival   = titre de film
